source/_integrations/zha.markdown

```diff
@@ -69,15 +69,17 @@ There is also support for Zigbee grouping of lights, switches, and fans (i.e. su
+Once ZHA has been set up and configured with a Zigbee Coordinator radio adapter or module you will be able to directly join/pair any Zigbee Router devices and Zigbee End Devices to the Zigbee network; [regardless of the manufacturer and brand of that Zigbee-based product](#knowing-which-devices-are-supported), (with few [limitations](#limitations)). While backward compatible with devices that use earlier versions of Zigbee Home Automation (ZHA) and Zigbee Light Link (ZLL) protocols, it is generally recommended to buy Zigbee 3.0 (ZB3) compliant devices. Zigbee 3.0 certified devices usually offer better interoperability with more Zigbee gateway solutions out-of-the-box, however [devices that use manufacturer-specific extensions to add non-standard functions and features will still need quirk code in custom device handlers to fully work with ZHA](#how-to-add-support-for-new-and-unsupported-devices).
 
-Once ZHA has been set up and the Zigbee Coordinator radio adapter or module is configured, you will be able to directly join/pair any Zigbee device to the Zigbee network; regardless of the manufacturer and brand of that Zigbee-based product. Note, that while it is generally recommended to buy Zigbee 3.0 compliant devices as those should, in theory, offer greater interoperability, be sure of the sections about which devices are supported and exception handling.
+Because a Zigbee relies on "mesh networking" technology it depends on its Zigbee Router devices for expanding the network coverage and size. These are always mains-powered devices that will act as Zigbee routers (often wrongly referred to as signal repeaters) within the Zigbee network mesh to extend its range and expand the total amount of devices that can be added. Zigbee End Device devices on the other hand will not expand the range of the network as they do not pass along any signals/messages from other devices, these are usually all battery-operated devices, but not always. You should therefore make sure that your Zigbee network has enough Zigbee Router devices, and always begin by adding those before adding Zigbee End devices when getting started.
+
+You will also keep in mind that a Zigbee network needs to have many Zigbee Router devices or else your network mesh coverage will be severely limited due to the short range and poor wall penetration of Zigbee radio signals. In addition, be aware that all Zigbee devices, and especially the Zigbee Coordinator, can be very sensitive to electromagnetic interference (EMI), As such it is highly recommended that you read and follow the best practice tips below about [Zigbee interference avoidance and network range/coverage optimization)](#zigbee-interference-avoidance-and-network-rangecoverage-optimization).
```

Comment:
I think this last paragraph is a little redundant, since we already have a section on RF interference.

Reply:
Think that at least the link should be keept as IMHO extremly important (probably solves more than half of all issues posted in Home Assistant community forum)

Reply:
Changed in https://github.com/home-assistant/home-assistant.io/pull/28869/commits/e9e308f0df0a4dc4eb1b27e706b27d7bfa7b7b1d